src/common/libdebugged/Makefile.am

```diff
@@ -11,11 +11,6 @@ AM_CPPFLAGS = \
 noinst_LTLIBRARIES = libdebugged.la
 libdebugged_la_SOURCES = debugged.c debugged.h
 
-libdebugged_la_CPPFLAGS = \
```

Comment:
Commit message: unnecessary apostrophe in `Makefile.am's`

Typo: "The caused the library" -> "This caused the library" ?